### PR TITLE
globby typings: use proper globby options

### PR DESCRIPTION
**Pull request by DomiR**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/sindresorhus/globby
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/globby/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "fast-glob": "^2.2.0"
```
Comment (ikatyang) [inline types/globby/package.json]: IIRC, we don't need to add this file.

File: types/globby/index.d.ts [hunk 2]
```diff
@@ -3,19 +3,20 @@
 // Definitions by: Douglas Duteil <https://github.com/douglasduteil>
```
Comment (ikatyang) [inline types/globby/index.d.ts]: We should bump the version to `// Type definitions for globby 8.0`.

File: types/globby/index.d.ts [hunk 3]
```diff
@@ -3,19 +3,20 @@
...
 
 declare namespace globby {
     /**
      * Returns an `Array` of matching paths.
      */
-    function sync(patterns: string | string[], options?: IOptions): string[];
+    function sync(patterns: string | string[], options?: Options): string[];
     /**
      * Returns an `Array<Object>` in the format `{ pattern: string, opts: Object }`,
      * which can be passed as arguments to [node-glob](https://github.com/isaacs/node-glob).
```
Comment (ikatyang) [inline types/globby/index.d.ts]: These looks outdated, we should also update the docs, just need to copy-paste from the [API](https://github.com/sindresorhus/globby#api).

**Review by ikatyang — CHANGES_REQUESTED** [review]

**Comment by DomiR** [comment]
thanks, yes the typing is more outdated than I thought...

**Pull request closed without merging**